UEFITool/CMakeLists.txt

```diff
@@ -13,6 +17,15 @@ SET(PROJECT_FORMS
  gotoaddressdialog.ui
  )
 
+SET(PROJECT_HEADERS 
```

Comment:
Why does only UEFITool have headers now and not UEFIExtract/UEFIFind, where they were explicitly removed?

Reply:
Turns out CMake is smart enough to derive all headers for non-Qt project, but not smart enough to derive a subset of Qt headers that require UIC pass to properly work, so some of those had to stay. The current list of header is the minimal one, i.e. if you remove any, CMake builds will start failing.